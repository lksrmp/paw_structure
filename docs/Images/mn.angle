ANGLE DISTRIBUTION FUNCTION
T1                7.49999173
T2               14.56325395
SNAPSHOTS              10000
ID1                       MN
ID2                       O_
CUT               3.50000000
NBINS                   1000
UNIT CELL     
   12.42959911     0.00000000     0.00000000
    0.00000000    12.42959911     0.00000000
    0.00000000     0.00000000    12.42959911

        DEGREE           ADF
    0.18000000     0.00000000
    0.36000000     0.00000000
    0.54000000     0.00000000
...
  179.64000000     0.00047880
  179.82000000     0.00051563
  180.00000000     0.00014732
