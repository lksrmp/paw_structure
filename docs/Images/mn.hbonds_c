HYDROGEN BONDS PER MOLECULE
T1                0.00012094
T2               14.56325395
SNAPSHOTS               1000
ID1                       O_
ID2                       H_
CUT1                     3.5
CUT2                     3.1
ANGLE                  140.0
UNIT CELL     
   12.42959911     0.00000000     0.00000000
    0.00000000    12.42959911     0.00000000
    0.00000000     0.00000000    12.42959911

          TIME HB / MOLECULE
    0.00012094     3.21875000
    0.01475519     3.21875000
    0.02926850     3.21875000
...
   14.53410640     3.46875000
   14.54861971     3.40625000
   14.56325395     3.59375000
