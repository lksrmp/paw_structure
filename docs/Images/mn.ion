ION COMPLEXES
T1                0.00012094
T2               14.56325395
SNAPSHOTS               1000
ID1                       MN
ID2                       O_
ID3                       H_
CUT1              3.00000000
CUT2              1.40000000
UNIT CELL     
   12.42959911     0.00000000     0.00000000
    0.00000000    12.42959911     0.00000000
    0.00000000     0.00000000    12.42959911
------------------------------------------------------------------------------------
TIME          0.00012094    ITERATION     1             ATOMS         19            
NAME          ID            INDEX                      X             Y             Z
MN0           MN            193               3.90303972    4.61530967    4.91739965
O_58          O_            175               4.28601969    5.22621963    6.99449950
O_59          O_            178               3.16676977    3.71929973    2.81496980
O_60          O_            181               5.08626964    2.72375981    4.54173967
O_61          O_            184               2.64793981    6.39776954    4.53175968
O_62          O_            187               2.14235985    3.36127976    5.44232961
O_63          O_            190               5.80626958    5.78752959    4.36323969
H_116         H_            117               3.60787974    5.41679961    7.67221945
H_117         H_            118               5.17293963    5.24571962    7.40457947
H_118         H_            119               2.17705984    3.69199974    2.58516981
H_119         H_            120               3.62565974    4.23226970    2.10257985
H_120         H_            121               5.96809957    2.85854980    4.95571965
H_121         H_            122               5.30523962    2.80249980    3.58775974
H_122         H_            123               2.54092982    7.12018949    5.16838963
H_123         H_            124               2.43533983    6.81989951    3.68914974
H_124         H_            125               1.59270989    3.70510973    6.18515956
H_125         H_            126               2.44026983    2.52562982    5.75588959
H_126         H_            127               6.34407955    6.21036956    5.04007964
H_127         H_            128               5.67375959    6.53968953    3.70842973
------------------------------------------------------------------------------------
TIME          0.01475519    ITERATION     122           ATOMS         19            
NAME          ID            INDEX                      X             Y             Z
MN0           MN            193               3.90303972    4.61530967    4.91739965
O_58          O_            175               4.28601969    5.22621963    6.99449950

...
