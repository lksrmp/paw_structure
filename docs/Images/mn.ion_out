ION COMPLEXES
T1                4.88324302
T2                5.05824928
SNAPSHOTS                  5
ID1           SEE ORIGINAL OUTPUT
ID2           SEE ORIGINAL OUTPUT
ID3           SEE ORIGINAL OUTPUT
CUT1              0.00000000
CUT2              0.00000000
UNIT CELL     
   12.42959911     0.00000000     0.00000000
    0.00000000    12.42959911     0.00000000
    0.00000000     0.00000000    12.42959911
------------------------------------------------------------------------------------
TIME          4.88324302    ITERATION     40376         ATOMS         18            
NAME          ID            INDEX                      X             Y             Z
MN0           MN            192               1.96977776    5.82753914    7.34376881
O_58          O_            174               1.94176345    3.75034659    8.28771089
...
H_126         H_            127               2.98893068    7.71062986    8.71133521
------------------------------------------------------------------------------------
TIME          4.89787727    ITERATION     40497         ATOMS         15            
NAME          ID            INDEX                      X             Y             Z
MN0           MN            192               1.97710096    5.80435805    7.34034399
O_58          O_            174               1.89558645    3.78353961    8.28209142
...
H_126         H_            127               2.96251970    7.82264169    9.02016312
------------------------------------------------------------------------------------
TIME          4.91239057    ITERATION     40617         ATOMS         18            
NAME          ID            INDEX                      X             Y             Z
MN0           MN            192               1.97506952    5.79277389    7.34848206
O_58          O_            174               1.83459064    3.80609580    8.25623914
...
H_126         H_            127               3.12739138    7.62703275    9.32214613
------------------------------------------------------------------------------------
TIME          5.04361503    ITERATION     41702         ATOMS         18            
NAME          ID            INDEX                      X             Y             Z
MN0           MN            192               1.94912018    5.86697344    7.36147804
O_58          O_            174               1.76759579    3.66340836    8.49068158
...
H_126         H_            127               3.55019817    7.07419308    9.09001855
------------------------------------------------------------------------------------
TIME          5.05824928    ITERATION     41823         ATOMS         15            
NAME          ID            INDEX                      X             Y             Z
MN0           MN            192               1.97889191    5.85306499    7.35932684
O_58          O_            174               1.70696083    3.61892158    8.45376365
...
H_126         H_            127               3.59765482    7.11809565    9.11958929
