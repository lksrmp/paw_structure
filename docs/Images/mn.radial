RADIAL DISTRIBUTION FUNCTION
T1                7.49999173
T2               14.56325395
SNAPSHOTS              10000
ID1                       MN
ID2                       O_
CUT               7.00000000
NBINS                   1000
RHO               0.03332795
UNIT CELL     
   12.42959911     0.00000000     0.00000000
    0.00000000    12.42959911     0.00000000
    0.00000000     0.00000000    12.42959911

        RADIUS           RDF  COORDINATION
    0.00700000     0.00000000     0.00000000
    0.01400000     0.00000000     0.00000000
    0.02100000     0.00000000     0.00000000
...
    6.98600000     0.83604464    47.97194550
    6.99300000     0.83018157    48.09126496
    7.00000000     0.96927922    48.22039411
