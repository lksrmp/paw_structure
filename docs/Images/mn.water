WATER COMPLEXES
T1                0.00012094
T2               14.61332485
SNAPSHOTS               1000
ID1                       O_
ID2                       H_
CUT               1.40000000
UNIT CELL     
   12.42959911     0.00000000     0.00000000
    0.00000000    12.42959911     0.00000000
    0.00000000     0.00000000    12.42959911
------------------------------------------------------------------------------------
TIME          0.00012094    ITERATION     1             ATOMS         6             
NAME          ID            INDEX                      X             Y             Z
H_26          H_            27                5.60287960    5.55340960    4.62225967
H_47          H_            48                2.03546985    0.50390996    5.92573958
H_52          H_            53                1.96003986    7.64422945    9.19131934
O_13          O_            147               5.37409962    6.44914954    4.80751966
O_24          O_            170               1.43365990    0.26245998    5.22945963
O_27          O_            188               1.22770991    6.95070950    9.12181935
------------------------------------------------------------------------------------
TIME          0.01475519    ITERATION     122           ATOMS         6             
NAME          ID            INDEX                      X             Y             Z
H_26          H_            27                5.60287960    5.55340960    4.62225967
H_47          H_            48                2.03546985    0.50390996    5.92573958
H_52          H_            53                1.96003986    7.64422945    9.19131934
O_13          O_            147               5.37409962    6.44914954    4.80751966
O_24          O_            170               1.43365990    0.26245998    5.22945963
O_27          O_            188               1.22770991    6.95070950    9.12181935
------------------------------------------------------------------------------------
TIME          0.02938944    ITERATION     243           ATOMS         6             
NAME          ID            INDEX                      X             Y             Z
H_26          H_            27                5.60287960    5.55340960    4.62225967

...
