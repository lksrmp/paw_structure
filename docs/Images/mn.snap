SELECTED SNAPSHOTS FROM TRAJECTORY FILE
T1                0.00012094
T2               14.56325395
SNAPSHOTS               1000
ATOMS                    193
UNIT CELL     
   12.42959911     0.00000000     0.00000000
    0.00000000    12.42959911     0.00000000
    0.00000000     0.00000000    12.42959911
------------------------------------------------------------------------------------
TIME          0.00012094    ITERATION     1             
NAME          ID            INDEX                      X             Y             Z
H_0           H_            1                 7.10222949    8.57026939    5.55092960
...
H_127         H_            128               5.67375959    6.53968953    3.70842973
O_8           O_            129               9.69160931    4.98455964    6.88164951
...
O_28          O_            192               4.29107969    0.92039993    0.63130995
MN0           MN            193               3.90303972    4.61530967    4.91739965
------------------------------------------------------------------------------------
TIME          0.01475519    ITERATION     122           
NAME          ID            INDEX                      X             Y             Z
H_0           H_            1                 7.10222949    8.57026939    5.55092960
...
H_127         H_            128               5.67375959    6.53968953    3.70842973
O_8           O_            129               9.69160931    4.98455964    6.88164951
...
O_28          O_            192               4.29107969    0.92039993    0.63130995
MN0           MN            193               3.90303972    4.61530967    4.91739965
------------------------------------------------------------------------------------

...
